app/components/atoms/social-media-buttons/social-media-button.component.tsx

```diff
@@ -0,0 +1,61 @@
+import {FC} from 'react';
+
+import facebookDark from './img/facebook-dark.svg';
+import facebook from './img/facebook.svg';
+import instagramDark from './img/instagram-dark.svg';
+import instagram from './img/instagram.svg';
+import twitterDark from './img/twitter-dark.svg';
+import twitter from './img/twitter.svg';
+import {Icon, StyledButton, Container} from './social-media-button.styles';
+
+export default interface Props {
```

Comment:
no need to export

Reply:
![image](https://user-images.githubusercontent.com/75398243/230242405-a23a2915-f249-416e-aa0e-fbe762c26ed1.png)
**But I use this export to import into the Styles File** 
![image](https://user-images.githubusercontent.com/75398243/230242474-2f5f1e38-728a-4bf9-a4d3-9aa4cfa03ba2.png)